ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -253,6 +253,11 @@ open class DrawerController: UIViewController {
             if presentationDirection.isHorizontal && resizingBehavior == .dismissOrExpand {
                 resizingBehavior = .dismiss
             }
+            
+            // If the drawer is already loaded, update the resizing handle when resizing behaviour is changed.
+            if self.isViewLoaded {
```

Comment:
shouldn't need self. also applies to other callers

Reply:
i will update this.